app/utils/cyaHelpers/item/ItemAnswersHelper.scala

```diff
@@ -34,10 +34,12 @@ import play.api.i18n.Messages
 import services.DocumentsService
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 import utils.cyaHelpers.AnswersHelper
+import viewmodels.Link
 
 // scalastyle:off number.of.methods
 class ItemAnswersHelper(
   userAnswers: UserAnswers,
+  mode: Mode,
```

Comment:
Don't need this as you're overriding the mode in AnswersHelper with CheckMode anyway